latitudesh/lsh
# Feat: Add label to optional fields in the interactive mode

## What

- Combine the flag and prompt declaration into one struct, for easier field declaration and maintenance
- Changes the required fields to be prompted first
- Add an "[Optional]” label to optional attributes in the prompt (interactive mode)
- Add a "[Required]” label to required attributes in the flags description (--help)
- Creates a flag to skip the interactive mode `--no-input`

## How to test

- Run commands that support interactive mode, with no flags, and validate if the [Optional] label is included for optional fields
- Run --help for a command and check if it will include a `[Required]` label for required fields
- Run a command with `--no-input`: it should skip the interactive mode
